smriprep/workflows/anatomical.py

```diff
@@ -221,8 +218,8 @@ def init_anat_preproc_wf(
 
     # 2. Brain-extraction and INU (bias field) correction.
     brain_extraction_wf = init_brain_extraction_wf(
-        in_template=skull_strip_template[0],
-        template_spec=skull_strip_template[1],
+        in_template=skull_strip_template.name,
```

Comment:
```suggestion
        in_template=skull_strip_template.fullname,
```

Reply:
In this case, we want just the name as the cohort is set via spec below.